spec/daemon_spec.rb

```diff
@@ -44,6 +44,10 @@
       wait_for_stop
       test_state(:server_restart_immediate).should == 1
 
+      dm.dump
```

Comment:
I'm not sure but I assume these lines cause test failure on windows.
Push new commits should re-run tests.